FTCBotCode/Competition24_25.java

```diff
@@ -1,25 +1,41 @@
 package org.firstinspires.ftc.teamcode;
 
-// import com.qualcomm.robotcore.hardware.hardwareMap;
 import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.hardware.ServoController;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
 
 @TeleOp(name = "FTC Competition (Blocks to Java)")
 public class Competition24_25 extends LinearOpMode {
-
+  //Drive
   private DcMotor frontRight;
   private DcMotor backRight;
   private DcMotor frontLeft;
   private DcMotor backLeft;
 
+  //Arm
+  private ServoController ControlHub_ServoController;
+  private Servo pincher;
+  private Servo wrist;
+  private DcMotor elbow;
+  private DcMotor lift
+  private DcMotor shoulder
+
   public void runOpMode() {
+    //Drive
     float horizontal;
     float vertical;
     float pivot;
-    float armDown;
-    float armUp;
+    double drivePower = 1;
+
+    //Arm
+    double liftPower = 1;
+    double shoulderPower = 1;
+    double elbowPower = 1;
```

Comment:
Just a small thing but I think that these should be changed to be private properties instead of local variables in case we start modularizing this code.

Reply:
I think we are going to keep them local for know, but we can change that later. I'm sure there will be a lot of future changes so we can add this on with those.